frontend/src/shared/form/form-item.vue

```diff
@@ -109,3 +109,9 @@ export default {
   name: 'AppFormItem',
 };
 </script>
+
+<style scoped>
+div.is-error {
+  margin-bottom: 40px;
+}
```

Comment:
Could we do this in the Activity form component?
I'm afraid that with this change, it will affect all form-items throughout the app which can cause some unexpected visual changes

Reply:
in the parent form div element for each field, the class appears twice or thrice in the case of the `isError` class
`<div class="el-form-item mb-4 mb-4">` or `<div class="el-form-item isError isError isError">`

any idea why this is happening?

Reply:
Interesting 🤔 
No, not sure why it is happening, also not being able to see it

Reply:
I'll send a screenshot when I can. Strange that's happening only for me. Although the code should as expected baring this occurrence, it doesn't really affect anything